src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -87,15 +90,15 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
         if (filtersHaveUpdated) {
           fetchResults()
 
-          // TODO: instrumentation?
-          // tracking.trackEvent({
-          //   action_type:
-          //     AnalyticsSchema.ActionType.CommercialFilterParamsChanged,
-          //   current: filterContext.filters,
-          //   changed: {
-          //     [filterKey]: filterContext.filters[filterKey],
-          //   },
-          // })
+          tracking.trackEvent({
+            context_page: AnalyticsSchema.PageName.ArtistAuctionResults,
+            action_type:
+              AnalyticsSchema.ActionType.AuctionResultFilterParamChanged,
```

Comment:
Might need @louislecluse to verify this, but I'm not sure we should create a new page type for this tab. I _think_ it should just live under the artist page's context.

Reply:
Forgot to link the slack discussion: https://artsy.slack.com/archives/C9YNS4X32/p1581631403052300?thread_ts=1581543838.020500&cid=C9YNS4X32

I might have misunderstood it though.

Reply:
Ahh, I see. All right then 👍